examples/v1beta1/pipelineruns/pipelinerun.yaml

```diff
@@ -246,13 +308,14 @@ spec:
   pipelineRef:
     name: demo-pipeline
   serviceAccountName: 'default'
-  resources:
-  - name: source-repo
-    resourceRef:
-      name: skaffold-git-pipelinerun
-  - name: web-image
-    resourceRef:
-      name: skaffold-image-leeroy-web-pipelinerun
-  - name: app-image
-    resourceRef:
-      name: skaffold-image-leeroy-app
+  workspaces:
+  - name: git-source
+    volumeClaimTemplate:
+      metadata:
+        name: temp-pvc-demo-pipeline-run-1
```

Comment:
I liked your proposal about omitting the name of `volumeClaimTemplate` in https://github.com/tektoncd/pipeline/issues/2450

I actually think we should omit the metadata and name part in our examples, it is more readable without and they don't provide much value. It is good to specify metadata when you want to add custom labels or annotations - but that is more advanced usage :)

Reply:
haha I left this PR open long enough that I can now use the feature you added @jlpettersson !!! XD thank you!!